xbmc/video/VideoDatabase.h

```diff
@@ -979,7 +979,7 @@ class CVideoDatabase : public CDatabase
     }
   }
 
-  void SetArtForItem(int mediaId, const MediaType &mediaType, const std::string &artType, const std::string &url);
+  void SetArtForItem(int mediaId, const MediaType &mediaType, const std::string &artType, const std::string &url) const;
```

Comment:
From the logical POV a setter methid being `const` looks somehow wrong.

Reply:
Only changed as Visual Studio/Resharper said it could be const - I thought the preference was for a function to be const where it could be?? Will change back.

Reply:
> I thought the preference was for a function to be const where it could be??

Yes, this is correct. I'm only irritated why a member function that does not change member's state can be const. This is from the logical perspective, not technical. Nothing you have introduced with your PR, and unrelated to your change, so I suggest to leave it as it is and to remove `const`.

Reply:
Removed.